classes/student.rb

```diff
@@ -2,6 +2,8 @@
 
```

Comment:
- [ ] As suggested by the previous reviewer kindly add belongs to side for student-classroom relationship. Means you should have the below method in your student class ⬇️

```ruby
  def classroom=(classroom)
    @classroom = classroom
    classroom.students.push(self) unless classroom.students.include?(self)
  end
```